implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -144,6 +144,23 @@ impl CreateCommand {
             ..cmd
         })
     }
+
+    fn parse_node_name(&self) -> crate::Result<Self> {
+        let cmd = self.clone();
+
+        if self.node_name.contains('/') {
+            if self.node_name.starts_with("/node/") {
+                let node_name = extract_address_value(&self.node_name)?;
+                return Ok(Self { node_name, ..cmd });
+            } else {
+                return Err(crate::Error::new(
+                    exitcode::IOERR,
+                    anyhow!("Error: A MultiAddr node must follow the format /node/{{name}}"),
+                ));
+            }
+        }
+        Ok(Self { ..cmd })
```

Comment:
This function should be static and should live in the utils.rs close to the `extract_address_value` and should return a `String` so we can reuse it elsewhere if needed. The input should be a `&str` too.

Reply:
Return types changed and function moved closer to `extract_address_value` for reusability in the latest commit